src/app/issuer/components/issuer-edit/issuer-edit.component.html

```diff
@@ -15,10 +15,13 @@
 		<fieldset>
 			<bg-formfield-image
 				#imageField
-				label="{{ 'Issuer.imageOptional' | translate }}"
+				label="{{ 'Issuer.imageLogo' | translate }}"
 				imageLoaderName="issuer"
 				[placeholderImage]="issuerImagePlacholderUrl"
 				[control]="issuerForm.rawControlMap.issuer_image"
+				[errorMessage]="imageError"
```

Comment:
This doesn't belong in this PR

Reply:
I intended this to be part of this PR, as i didnt want to open another ticket for it. I updated the PR description

Reply:
I don't think this makes much sense, especially as the changes could have been separated quite easily. The ticket this PR refers to only needs a style change after all. But maybe this is a discussion that won't be easily resolved in those comments.

Reply:
No i agree, i split up the PRs now